e2e_tests/test_admin.py

```diff
@@ -172,3 +172,11 @@ def test_should_not_allow_move_when_same_case_ignored_title_in_same_sector(
     )
 
     page.close()
+
+
+def test_should_allow_local_groups_without_image(live_server, page: Page):
+    admin_login(live_server.url, page)
+    page.goto(live_server.url + "/admin/cpmonitor/city/2/change/")
+    page.fill("[name=local_group-0-name]", "OhnenixZero")
+    page.click("[name=_save]")
```

Comment:
I prefer to use things that are visible to the user in this case `page.get_by_role("button", name="Sichern", exact=True).first.click()` because then the test is easier to understand and resembles more closely what the user does.